Notebooks/Simulations/skypy_lenstronomy.ipynb

```diff
@@ -29,7 +29,7 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.            ax.imshow(img, aspect='equal', origin='lower')
</pre><p>One thing that I generally wonder is whether we can get more colorful pictures showing more colors from blue, green over yellow to red</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>